tyk-pro/README.md

```diff
@@ -2,32 +2,43 @@
 Provides a full Tyk Installation API Management Dashboard and API Gateways with Analytics)
 
 ### Installation
-To install, *first modify the `values.yaml` file and add your license*. Then run the following command from the root of the repository:
-
-	helm install tyk-pro ./tyk-pro -n tyk --wait
-
+```
+helm repo add tyk-helm https://helm.tyk.io/public/helm/charts/
+helm repo update
+```
+Before we proceed with installation of the chart we need to set some custom values. To see what options are configurable on a chart and save that options to a custom values.yaml file run:
+```
+helm show values tyk-helm/tyk-pro > values.yaml
+```
+*For Tyk-pro chart we need to modify following value in your custom values.yaml file:*
+1. Add your dashboard license in `dash.license` value
+
+Then we can install the chart using our custom values:
+```
+helm install tyk-pro tyk-helm/tyk-pro --version 0.9.0 -f values.yaml -n tyk --wait
+```
 > Please note the --wait argument is important for successful dashboard bootstrap.
 
 Follow the instructions in the Notes that follow the installation to find your Tyk login credentials.
 
 ### Tyk Developer Portal
-You can enable the bootstraping of the Developer Portal using the `portal.bootstrap` option in `values.yaml`.
+You can enable the bootstrapping of the Developer Portal using the `portal.bootstrap` option in `values.yaml`.
```

Comment:
@vesko-tyk change it to "disable" since it's on by default

Reply:
Thanks @letzya. This is now updated on master.